test/integration/multiple.done.js

```diff
@@ -16,7 +16,7 @@ describe('multiple calls to done()', function() {
 
     it('results in failures', function() {
       assert.equal(res.stats.pending, 0);
-      assert.equal(res.stats.passes, 1);
+      assert.equal(res.stats.passes, 0);
```

Comment:
Ah sorry for quoting the wrong fixture.

I actually looked at the commented out section of that fixture, but didn't know what the the correct behaviour is.

If I uncomment that fixture, the change to the test to make it pass looks like this:
```patch
     it('results in failures', function() {
       assert.equal(res.stats.pending, 0);
-      assert.equal(res.stats.passes, 0);
-      assert.equal(res.stats.failures, 1);
-      assert.equal(res.code, 1);
+      assert.equal(res.stats.passes, 1);
+      assert.equal(res.stats.failures, 2);
+      assert.equal(res.code, 2);
     });
```

Which seems strange since the `beforeEach` error should cancel the execution of the test, but doesn't do that (I confirmed that the test is executed) But this could be an acceptable limitation of the async beforeEach implementation. If you clarify if this is acceptable I can uncomment it in my PR.